app/uk/gov/hmrc/eoricommoncomponent/frontend/views/layout.scala.html

```diff
@@ -72,9 +72,6 @@
         @styles_di()
         @partials.countrypickerscript(countriesInCountryPicker, journey)
 
-        @if(isLoggedIn){
-        <noscript><meta http-equiv="refresh" content="900;url=@signOutUrl" /></noscript>
-        }
```

Comment:
Do you know why we had this? Was it a requirement?

Reply:
Or is it change anything in the page behaviour?

Reply:
We inherited this.   It is for non-javascript users - an attempt to automatically reloaded the page they are on (to try and prevent session timeout) even if the user is v. slow and is in the middle of editing/adding information.

I just removed it as requested.

Reply:
Thanks for clarification